src/plugins/discover/public/application/main/components/sidebar/lib/visualize_trigger_utils.ts

```diff
@@ -52,6 +52,7 @@ export function triggerVisualizeActions(
     indexPatternId,
     fieldName: field.name,
     contextualFields,
+    originatingApp: 'discover:sidebar',
```

Comment:
I don't think it is a good idea to call this "originatingApp" and then include more information about like "sidebar". How about changing to just "discover"? Also is there a constant for "discover"